pytket/tests/circuit_test.py

```diff
@@ -81,6 +82,16 @@
 from pytket.pauli import Pauli
 from pytket.transform import PauliSynthStrat, Transform
 
+if TYPE_CHECKING:
+    from pytket.circuit.named_types import (
```

Comment:
Already imported unconditionally above; not sure why we need this?

Reply:
I have removed the other import.